### PR TITLE
add first task

### PR DESCRIPTION
**Pull request by AndreyShushket**

### CODE REVIEW

File: Mentoring/Mentoring/Program.cs [hunk 1]
```diff
@@ -1,15 +1,20 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace Mentoring
+namespace ConsoleApp2
```
Comment (VladimirVolkovVD) [inline Mentoring/Mentoring/Program.cs:3]: Old name was better

**Review by VladimirVolkovVD — APPROVED** [review]
ok

File: Mentoring/Mentoring/Program.cs (hunk 1 above)
Comment (AndreyShushket) [reply Mentoring/Mentoring/Program.cs:3]: fuck off cunt

Comment (AndreyShushket) [reply Mentoring/Mentoring/Program.cs:3]: ok,thx

**Pull request merged**